### ISSUE TITLE
After giving my . aep file to export . json, the progress bar is full, but where there is no finish button, there is only a cancel button.

### ISSUE THREAD

**Issue by kabirshshryar** [airbnb/lottie-web#2431]
Here is the aep file - https://drive.google.com/file/d/1h1ZHlhIFNpmGdNt7k7VDo4nL0VF7Z0Sq/view?usp=drivesdk

**Comment by bodymovin** [comment]
Hi, can you share a screenshot of what you're seeing when render is done?

**Comment by kabirshshryar** [comment]
> Hi, can you share a screenshot of what you're seeing when render is done?

![fc878970b83465c17260a56c6a32c08a 0](https://user-images.githubusercontent.com/41082143/104843030-c972b580-58f2-11eb-8474-745d73e1681e.jpeg)

**Comment by henriqrossatto** [comment]
Hi @kabirshshryar

I got export your file. Not so sure but can be some bug related to images. As you are working with psd and png you have to wait render all the images. Try to work with shape layers inside the AE. If you have an AI file, convert for shape layers too and take care with merge paths or artboards that can be important. Mainly if you are working with gradient.

Attached the zip file with the json exportation. I export the json with the images inside them.


[data_2431.zip](https://github.com/airbnb/lottie-web/files/5827001/data_2431.zip)

**Comment by bodymovin** [comment]
Can you try using version 5.7.6? The issue should be fixed.

**Comment by henriqrossatto** [comment]
selected settings
![image](https://user-images.githubusercontent.com/30601233/105166502-46ba4800-5af6-11eb-8ef9-adcf52d09f3c.png)

fail render
![image](https://user-images.githubusercontent.com/30601233/105166574-59cd1800-5af6-11eb-8e17-c50ac76b5f32.png)

New comp created after the render
![image](https://user-images.githubusercontent.com/30601233/105166721-841ed580-5af6-11eb-8c9c-e44c3cff89df.png)

**Comment by bodymovin** [comment]
@henriqrossatto once the render fails, can you try going in your browser to http://localhost:8092/
then on the right panel selecting Console
and sharing a screenshot of that view?

**Comment by henriqrossatto** [comment]
@bodymovin here the screenshot

![image](https://user-images.githubusercontent.com/30601233/105560143-cbbd8100-5cf1-11eb-81e4-fb5747f230b0.png)

**Comment by bodymovin** [comment]
@henriqrossatto you're getting an error related to Chrome's API, but not related to this problem.
There is another way more complex way of doing this:
In Chrome, you'll need to go to
chrome://inspect/#devices
click the Configure button, and add an item `localhost:8092`
then select Done
after some seconds, if bodymovin is open in AE, you should see an element on the list below `Remote Target`.
click on inspect, and then take another screenshot of the new panel that opens, on the Console tab.

**Comment by henriqrossatto** [comment]
Like this?
![image](https://user-images.githubusercontent.com/30601233/105560816-b47f9300-5cf3-11eb-873c-e34c39668545.png)

**Comment by bodymovin** [comment]
@henriqrossatto yes, thanks. It seems to be an issue with the position of the audio on the timeline. It might be related to having an audio layer that starts before the zero time of the composition, or some rounding problem when moving it to a new composition. If you can share the .aep I'll take a look and try to have a fix soon.
In the meantime, you can try moving the audio layer in the timeline and see if it gets fixed.

**Comment by henriqrossatto** [comment]
Yes, this time works. 

Now what should I do to test it with the sound?

**Comment by henriqrossatto** [comment]
Hi @bodymovin any thoughts?

**Comment by bodymovin** [comment]
the easiest way is to add the howler library to the global scope.
Did you try any of these options?
https://github.com/airbnb/lottie-web/wiki/Audio

**Comment by henriqrossatto** [comment]
Hmm... I don't so familiar with this code part, I work in motion part.

If I try to adjust it inside the demo.html it can work?

**Comment by bodymovin** [comment]
yes, it should work adding this line at the beggining
`<script>https://cdnjs.cloudflare.com/ajax/libs/howler/2.2.0/howler.min.js</script>`
also keep in mind that browsers request user input to play sounds, so you might need to click anywhere on the screen

**Comment by henriqrossatto** [comment]
Ok. I will test it. Thank you!

Sorry to disturb you again but I'm with another problem. If I export with an audio, when I open the demo.html there is nothing in the screen but when I export withou the audio, the animation appear

**Comment by bodymovin** [comment]
@henriqrossatto can your .aep with the related files?

**Comment by henriqrossatto** [comment]
Yes, here
[lottie_error.zip](https://github.com/airbnb/lottie-web/files/5937573/lottie_error.zip)
